code/game/machinery/vending.dm

```diff
@@ -1589,5 +1601,241 @@
 	if(choice)
 		SetName(choice)
 
-
+/obj/machinery/vending/Merc_clothing_vendor
+	name = "Merc'ury Clothing Solutions"
+	desc = "Your one Stop Shop for Surplus Military Equipment"
+	product_slogans = "Go gettem!"
+	product_ads = "YOU WANNA LIVE FOREVER!; One hundred percent Kevlar!; Not rated for giant Mantas Invasions.; Market research shows that flak vests will replace shirts entirely in a few years. Get ahead of the fashion!"
+	icon_state = "serbomat" // Placeholder till I can make a new one -R
+	no_criminals = FALSE
+	products = list( 
+					/obj/item/clothing/under/serbiansuit = 4,
+					/obj/item/clothing/under/serbiansuit/brown = 4,
+					/obj/item/clothing/under/serbiansuit/black = 4,
+					/obj/item/clothing/head/soft/green2soft = 4,
+					/obj/item/clothing/head/soft/tan2soft = 4,
+					/obj/item/clothing/suit/armor/flak = 2,
+					/obj/item/clothing/suit/armor/flak/green = 2,
+					/obj/item/clothing/suit/armor/flak/full = 2,
+					/obj/item/clothing/suit/armor/flak/full/green = 2,
+					/obj/item/clothing/suit/armor/bulletproof/serbian = 2,
+					/obj/item/clothing/suit/armor/bulletproof/serbian/green = 2,
+					/obj/item/clothing/suit/armor/bulletproof/serbian/tan = 2,
+					/obj/item/clothing/suit/armor/bulletproof/serbian/full = 2,
+					/obj/item/clothing/suit/armor/bulletproof/serbian/full/green = 2,
+					/obj/item/clothing/suit/armor/bulletproof/serbian/full/tan = 2,
+					/obj/item/clothing/head/armor/helmet = 4 
```

Comment:
@Jamini I'm sure you've already looked, but I would be remiss if I didn't double-check that these armors are balanced for player use in these quantities!

Reply:
Hes looked at the prices, but not amounts. There are x2 Vendors, so that has double the armors around